package.json

```diff
@@ -5,7 +5,8 @@
   "main": "server/App.js",
   "scripts": {
     "start": "node server/App.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/.bin/eslint server/"
```

Comment:
You can change this to `eslint server/` since `npm` automatically has `node_modules/.bin/` in it's path.

Reply:
but locally its not in general I think? Unless you have eslint globally of course

Reply:
@vatolvan no, that holds true for all `npm` installations